pandas/core/tools/datetimes.py

```diff
@@ -492,12 +492,12 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
           origin.
 
         .. versionadded:: 0.20.0
-    cache : boolean, default False
+    cache : boolean, default True
         If True, use a cache of unique, converted dates to apply the datetime
         conversion. May produce significant speed-up when parsing duplicate
         date strings, especially ones with timezone offsets.
 
-        .. versionadded:: 0.23.0
+        .. versionadded:: 0.25.0
```

Comment:
Can you leave the versionadded in there, but also add a ``.. versionchanged:: 0.25.0`` ?

Reply:
@jorisvandenbossche I did it. Can you see and evaluate?

Reply:
Yes, that looks good, thanks. I just made a punctuation suggestion.

Reply:
Thanks @jorisvandenbossche . Suggestion is accepted.